test/utils/bytecode.hpp

```diff
@@ -64,6 +69,35 @@ inline bytecode operator*(int n, evmc_opcode op)
     return n * bytecode{op};
 }
 
+inline bytes big_endian(uint16_t value)
+{
+    return {static_cast<uint8_t>(value >> 8), static_cast<uint8_t>(value & 0xff)};
+}
+
+inline bytecode eof_header(uint8_t version, uint16_t code_size, uint16_t data_size)
+{
+    bytecode out{"ef00"};
+    out += bytes{version};
```

Comment:
This works as `bytecode{bytes{0xEF, 0x00, version}}`, because `bytecode` doesn't have `initializer_list` constructor. And adding such constructor would break `bytecode{int_const}` expressions, which currently translates to PUSH.